Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -56,7 +56,7 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
     )
     if threshold is not None:
         assign(state, I > threshold)
-    return state, I, Q
+    return state
 
```

Comment:
Sometimes you need the I & Q, so this needs to stay, we need to fix the scripts.
I actually have a PR which fixes this from about two weeks ago that wasn't merged :\

Reply:
ok, well how do we proceed? I  get confused with all of these github methodology now xD

Reply:
I'll help you rebase it on main on Monday

Reply:
Try to address the rest of the comments here and next week we'll do the github stuff together.